modules/Microsoft.Network/dnsForwardingRulesets/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,84 @@
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the DNS Resolver to create.')
+param dnsResolverName string
+
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+  name: virtualNetworkName
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.10.100.0/24'
```

Comment:
In alignment with other modules that deploy networks (e.g., key vault common) I'd suggest to uses variables for the address spaces as well as similar address spaces. Just for consistency